load.php

```diff
@@ -79,29 +79,56 @@ function perflab_render_generator() {
 add_action( 'wp_head', 'perflab_render_generator' );
 
 /**
- * Gets the standalone plugin constants used for each available standalone plugin.
+ * Gets the standalone plugins recommended by Performance Lab and their data.
  *
- * @since 2.9.0
- * @since n.e.x.t The $source parameter was removed.
+ * @since n.e.x.t
  *
- * @return array<string, string> Map of plugin slug and the version constant used.
+ * @return array Associative array of $plugin_slug => $plugin_data pairs.
  */
-function perflab_get_standalone_plugin_version_constants() {
```

Comment:
Note for reviewers: This function is simply moved down (see bottom of the file), it's not being replaced as potentially the diff here may suggest.